code/game/objects/items/stacks/sheets/glass.dm

```diff
@@ -270,6 +270,12 @@
 	var/cooldown = 0
 	sharpness = IS_SHARP
 
+/obj/item/weapon/shard/attackby(obj/item/S, mob/user, params)

```

Comment:
I personally prefer attackby but maintainers want tablecrafting in cases like this